src/test_util.rs

```diff
@@ -61,6 +61,40 @@ macro_rules! assert_lint_err {
       $($src: $test,)*
     }
   };
+
+  (
+    $rule: ty,
+    $message: expr,
+    $hint: expr,
+    filename: $filename:literal,
+    $($src:literal : $test:tt),+
+    $(,)?
+  ) => {
+      $(
+      let errors = parse_err_test!($message, $hint, $test);
+      let tester = $crate::test_util::LintErrTester::<$rule>::new(
+        $src,
+        errors,
+        $filename,
+      );
+      tester.run();
+    )*
+  };
+  (
+    $rule: ty,
+    $message: expr,
+    $hint: expr,
+    $($src:literal : $test:tt),+
+    $(,)?
+  ) => {
+      assert_lint_err! {
+        $rule,
+        $message,
+        $hint,
+        filename: "deno_lint_err_test.ts",
+        $($src: $test,)*
+      }
+  };
```

Comment:
```suggestion
  ) => {
    assert_lint_err! {
      $rule,
      $message,
      $hint,
      filename: "deno_lint_err_test.ts",
      $($src: $test,)*
    }
  };
```
This is also about formatting. Since Rust codes in Deno adopt 2-space indent, it'd be better to follow it in a macro.

Reply:
Sure, it's my bad that I did not follow the convention. I will modify and update the PR. Thank you for suggesting the changes.